### PR TITLE
[GSoC] Refactored rv and rv_interface

### PR DESCRIPTION
**Pull request by ritesh99rakesh**
#### References to other Issues or PRs

Removed unused imports and other minor changes

#### Brief description of what is fixed or changed
Added kurtosis function in `sympy/stats/rv_interface.py`
and added tests for different distributions.

#### Other comments
I deleted my previous fork of `sympy` and hence reopening this PR.
ping @sidhantnagpal @oscargus 

#### Release Notes

NO ENTRY

### CODE REVIEW

**Comment by ritesh99rakesh** [comment]
@sidhantnagpal Please review this.

File: sympy/stats/tests/test_rv.py [hunk 1]
```diff
@@ -181,7 +179,8 @@ def test_dependent_finite():
 
 def test_normality():
     X, Y = Normal('X', 0, 1), Normal('Y', 0, 1)
-    x, z = symbols('x, z', real=True, finite=True)
+    x = Symbol('x', real=True, finite=True)
+    z = Symbol('z', real=True, finite=True)
```
Comment (sidhantnagpal) [inline sympy/stats/tests/test_rv.py]: Was the former problematic?

Comment (ritesh99rakesh) [reply sympy/stats/tests/test_rv.py]: I was able to remove to remove the import of `symbols` as it was used here only.

File: sympy/stats/rv_interface.py [hunk 2]
```diff
@@ -1,5 +1,3 @@
-from __future__ import print_function, division
```
Comment (sidhantnagpal) [inline sympy/stats/rv_interface.py]: These imports are generally kept.

Comment (ritesh99rakesh) [reply sympy/stats/rv_interface.py]: Done. Thanks!

**Pull request merged**